@uportal/portlet-registry-to-array/package.json

```diff
@@ -67,5 +67,8 @@
   },
   "engines": {
     "node": "^6.0.0 || ^8.0.0 || ^10.0.0"
+  },
+  "dependencies": {
+    "lodash.uniqby": "^4.7.0"
```

Comment:
Could the `package-lock.json` be updated to reflect this change?
running `npm install` with npm version 5.2 or higher should do the trick.